libs/core/src/lib/shellbar/user-actions-menu/user-actions-menu.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { IconModule } from '../../icon/public_api';
+import { PopoverModule } from '../../popover/public_api';
+import { ListModule } from '../../list/public_api';
+import { TitleModule } from '../../title/public_api';
+import { AvatarModule } from '../../avatar/public_api';
+import { DialogModule } from '../../dialog/public_api';
+import { ButtonModule } from '../../button/public_api';
+
+import { UserActionsMenuComponent } from './user-actions-menu/user-actions-menu.component';
+import { UserActionsMenuItemComponent } from './user-actions-menu-item/user-actions-menu-item.component';
+
+@NgModule({
+  declarations: [UserActionsMenuComponent, UserActionsMenuItemComponent],
+  exports: [UserActionsMenuComponent, UserActionsMenuItemComponent],
+  imports: [
+    CommonModule,
+    PopoverModule,
+    IconModule,
+    AvatarModule,
+    ListModule,
+    TitleModule,
+    DialogModule,
+    ButtonModule
+  ]
+})
+export class UserActionsMenuModule { }
```

Comment:
Can you please add `UserActionsMenuModule` to the `FundamentalNgxCoreModule`?

Reply:
If we add to `FundamentalNgxCoreModule`, we should create separate doc and examples for this. But for now it's only a part of shellbar module.

Reply:
oh, ok, thanks for the explanation